integration-tools/src/main/java/uk/ac/bbsrc/tgac/miso/integration/dp5mirage/DP5MirageScanner.java

```diff
@@ -65,64 +68,65 @@ public BoxScan getScan() throws IntegrationException{
       // JSON response from Scanner
       response = GetPostParamRequest(httpClient, uri, DP5MirageScanner.params);
 
-      String errorMessage = "Error reported by the scanner.";
-      String actionMessage = "please check this before rescanning.";
-
       // Check if valid before parsing
       if (response.statusCode() != 200) {
         if (response.statusCode() == 453) {
-          throw new IntegrationException(String.format(" %s Container is not found. Check that a "
-              + "container with the"
-              + " container ID: %s is created on the DP5 application", errorMessage,
-              DP5MirageScanner.params.get(
-              "container_uid")));
+          throwScanError(String.format("Container is not found. Check that a container with the "
+                  + "container ID: %s is created on the DP5 application", DP5MirageScanner.params.get(
+                      "container_uid")),
+              true);
         }
         else if (response.statusCode() == 456) {
-          throw new IntegrationException(String.format("%s Scanner type mismatch. %s",
-              errorMessage, actionMessage));
+          throwScanError("Scanner type mismatch", false);
         }
         else if (response.statusCode() == 459) {
-          throw new IntegrationException(String.format("%s The scan is not found. Try rescanning "
-              + "after a short period of time.", errorMessage));
+          throwScanError("The scan is not found. Try rescanning after a short period of time",
+              true);
         }
         else if (response.statusCode() == 461) {
-          throw new IntegrationException(String.format("%s Scanner not connected. %s",
-              errorMessage, actionMessage));
+          throwScanError("The Scanner is not connected. Please check the connection before "
+              + "rescanning",true);
         }
         else if (response.statusCode() == 468) {
-          throw new IntegrationException(String.format("%s Scan result is not ready. Try "
-                  + "rescanning after a short period of time.",
-              errorMessage));
+          throwScanError("Scan result is not ready" ,false);
         }
         else if (response.statusCode() == 477) {
-          throw new IntegrationException(String.format("%s Failed to read a barcode. %s",
-              errorMessage, actionMessage));
+          throwScanError("Failed to read a barcode" ,false);
         }
         else if (response.statusCode() == 478) {
-          throw new IntegrationException(String.format("%s Scanner not found. %s", errorMessage,
-              actionMessage));
+          throwScanError("Scanner not found" ,false);
```

Comment:
We had discussed this message being sent to the user. I'm not entirely sure what it means since it's different from "scanner not connected", but it may be helpful